### PR TITLE
Run tests under wasmtime as part of the CI

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -35,15 +35,23 @@ jobs:
...
           # Upload the dist folder. Give it a name according to the OS it was built for.
           name: ${{ format( 'dist-{0}', matrix.os) }}
           path: dist
+      - name: Compile tests
+        run: make check
+        if: matrix.os == 'ubuntu-latest'
+      - name: "Run tests: wasmtime"
+        run: |
+          VERSION=v0.8.0
+          wget https://github.com/bytecodealliance/wasmtime/releases/download/v0.8.0/wasmtime-$VERSION-x86_64-linux.tar.xz
```
Comment (pchickey) [inline .github/workflows/main.yml:50]: additional $VERSION interpolation in url here

**Comment by TerrorJack** [comment]
@sbc100 I have patches in my `wasi-sdk` fork that fix the tests and run them via `wasmtime`; if you don't mind, I'd like to open another PR to port my patches?

**Comment by sbc100** [comment]
> @sbc100 I have patches in my `wasi-sdk` fork that fix the tests and run them via `wasmtime`; if you don't mind, I'd like to open another PR to port my patches?

Hmm.. this PR is 2.5 years old!  Please feel free to replace it with something newer.

File: .github/workflows/main.yml (hunk 1 above)
Comment (radical) [inline .github/workflows/main.yml:50]: Maybe join these commands with `&&` so we don't continue if the previous one fails.

**Comment by TerrorJack** [comment]
Shall we close this one given #278 has landed for some time?

**Pull request closed without merging**